untl/README.rst

```diff
@@ -0,0 +1,431 @@
+
+The UNT Libraries' metadata element set comprises Dublin Core-based descriptive metadata and detailed technical and preservation information recording how digital resources are identified, created, formatted, arranged in relevant software applications, and sustained with application of appropriate preservation procedures. While promoting interoperability with widely accepted standards, the recommended UNT Libraries' metadata elements allow flexibility at the local level to integrate with existing and anticipated content, processes, and systems.
+
+The UNT Libraries' Metadata Initiative progressed on many fronts. This (version-3.0) guideline is a product of a series of revision activities. Version-3.0 is indeed, a major revision of the UNTL metadata guidelines that offers many new and improved features, including major changes in the handling of sub-elements and qualifiers or refinments. It should be noted that in order to comply with evolving internal and external requirements, the UNT Libraries' metadata creation guideline will remain under continuous review. 
+
+An older version of the specification of all UNTL metadata elements and related maintanance activities, including element refinements, mapping, and controlled terms, available at: http://www.library.unt.edu/digitalprojects/metadata
+
+If you have any comments or suggestions regarding the metadata elements, please contact Daniel Gelaw Alemneh at daniel.alemneh@unt.edu.
+
+Element Reference Guide
+-----------------------
+
+Properties of the UNTL data dictionary discriptions:
+
+:Element: Name of element that is used in the machine-readable encoded document. 
+:Label: Human friendly version of the element name. The full name of the element is usually a word or phrase that identifies the element's purpose. In the tag library, the element name follows the tag name on the page defining that element and appears with initial capital letters, e.g.,
+:Sub Elements: Sub elements an element has. If an element lists its sub-elements, each sub-element has its own entry and gets the same treatment as an element in its description.
+:Definition: A brief statement that represents or defines the element
+:Comment:  Additional information about the element, including intended use or how values may be obtained.
+:Obligation: Indicates whether or not the element value is mandatory: "Required" or "Not required".
+:Repeatability: Indicates whether or not the element can take multiple values: "Repeatable" or "Not repeatable".
```

Comment:
Do you guys think this is a good description? Perhaps it should be changed to something like "Indicates whether or not the element can appear multiple times to indicate different values." or something along those lines?